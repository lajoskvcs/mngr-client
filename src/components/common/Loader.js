import React, {PropTypes} from 'react';
import 'loaders.css';
class Loader extends React.Component {
    render() {
        return (
            <div className="loader floating-centered">
                <div className="ball-grid-pulse">
                    <div />
                    <div />
                    <div />
                    <div />
                    <div />
                    <div />
                    <div />
                    <div />
                    <div />
                </div>
            </div>
        );
    }
}



export default Loader;