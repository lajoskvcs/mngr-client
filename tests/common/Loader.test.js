import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../../components/common/Loader';

test('Loader should render', () => {
  const component = renderer.create(
    <Loader />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});