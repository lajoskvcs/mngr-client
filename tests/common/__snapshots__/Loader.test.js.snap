// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render 1`] = `
<div
  className="loader floating-centered"
>
  <div
    className="ball-grid-pulse"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
